content/en/synthetics/platform/private_locations/_index.md

```diff
@@ -1021,6 +1021,7 @@ Use [granular access control][24] to limit who has access to your test based on
 7. Click **Done**.
 
 <div class="alert alert-info">You can view results from a Private Location even without Viewer access to that Private Location.</div>
+<div class="alert alert-info">Restricting a Private Location may limit other users from adding it to a test or editing it, but they are still able to see the location's name if it was added to a test by an authorized user.</div>
```

Comment:
I don't know if adding two consecutives `Note` section is allowed or if there is a better way to do that?